src/riak_kv_app.erl

```diff
@@ -155,6 +155,10 @@ start(_Type, _StartArgs) ->
                                           [true, false],
                                           false),
 
+            riak_core_capability:register({riak_kv, w1c_batch_vnode},
```

Comment:
Rolling upgrades. If a batch is sent to a 1.3 node, it must not deliver batches to 1.2 nodes in the cluster.